pkgs/default.nix

```diff
@@ -310,8 +310,8 @@ in rec {
 
   thumbor = (import ./thumbor { inherit pkgs newrelic-python statsd; }).thumbor;
 
-  upcast = pkgs.haskellPackages.callPackage ./upcast {
-    inherit awsEc2 vkPosixPty vkAwsRoute53;
+  upcast = pkgs.haskellngPackages.callPackage ./upcast {
+    inherit vk-posix-pty;
```

Comment:
is there a reason to keep vk-posix-pty here though? it's already in nixpkgs-14.12